internal/cmd/options.go

```diff
@@ -15,3 +19,24 @@ func (opts *SignerFlagOpts) ApplyFlags(fs *pflag.FlagSet) {
 	SetPflagKey(fs, &opts.Key)
 	SetPflagSignatureFormat(fs, &opts.SignatureFormat)
 }
+
+// Common option struct.
+type Common struct {
+	Debug bool
+}
```

Comment:
The naming `Common` is a bit too vague. Can we have something similar to the `SignerFlagOpts ` above?

Reply:
Renamed it to `LoggingFlagOpts`